contracts/CodexRecordMetadata.sol

```diff
@@ -32,6 +32,18 @@ contract CodexRecordMetadata is ERC721Token {
   //  via the tokenURI method
   string public tokenURIPrefix;
 
+  /**
+   * @dev Modifier that checks to see if the token exists
+   * @param _tokenId uint256 The token ID
+   */
+  modifier tokenExists(uint256 _tokenId) {
+    require(
+      exists(_tokenId),
+      "Token doesn't exist");
+
+    _;
```

Comment:
I don't think we need the extra line space here. I don't see it in other modifiers.